bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -89,7 +89,10 @@ def _execute_integtest_sh(self, endpoint, port, security):
         )
         if os.path.exists(script):
             cmd = f"{script} -b {endpoint} -p {port} -s {str(security).lower()} -v {self.bundle_manifest.build.version}"
-            (status, stdout, stderr) = execute(cmd, self.repo.dir, True, False)
+            if self.test_config.working_directory is not None:
```

Comment:
Nit: because these are the same operation functionally that differ in parameters I would:

```python
cmd = self.repo.dir
if self.test_config.working_directory:
   cmd = os.path.join(cmd, self.test_config.working_directory)
```

or

```
cmd = os.path.joimn(self.repo.dir, self.test_config.working_directory ) if self.test_config.working_directory else self.repo.dir
```

or, if `os.path.join(..., None)` works just `cmd = os.path.join(self.repo.dir, self.test_config.working_directory)`.